custom_typings/yeoman-environment.d.ts

```diff
@@ -4,7 +4,7 @@ declare module 'yeoman-environment' {
     registerStub(generator: any, namespace: string);
     run(target: string, options: any, done: Function);
     getGeneratorsMeta(): YeomanEnvironment.GeneratorMeta[];
-    lookup(callback: () => void);
+    lookup(callback: (error?: Error) => void);
```

Comment:
In a point release yeoman-environment would be in their rights to pass anything in as an err value though. I still recommend `any`